src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
```

Comment:
let's do this instead:

```suggestion
import React, {useState} from 'react';
```